### DISCUSSION TITLE
Are multiline comments available for free users?

### DISCUSSION THREAD

**Discussion by Nostromos** [community/community#63952] [New to GitHub]
The docs say they are but for the life of me I can't get them to work, despite lots of searching and trying on multiple browsers.

**Comment by spenserblack** [comment]
Is this what you're talking about? https://github.com/Nostromos/Jammming/pull/16/files#r1296208208

I'm a free user, and can definitely make multiline comments. I hover my mouse somewhere between the line number and the code in the diff view, and a `+` icon pops up that I can click and drag.

**Comment by Nostromos** [comment]
Exactly. For the last few hours, I was not able to make multiline comments
either clicking + drag or highlighting line numbers + click comment button. It
wasn't working on Chrome, Safari, or Firefox in normal or incognito mode for any
of them. Whatever the issue was seems to have resolved  for my account so I'll
close the topics I posted. Thanks for engaging with the question.  





On Wed, Aug 16, 2023 at 1:12 PM Spenser Black ***@***.***>  wrote:


Is this what you're talking about? 
https://github.com/Nostromos/Jammming/pull/16/files#r1296208208

I'm a free user, and can definitely make multiline comments. I hover my mouse
somewhere between the line number and the code in the diff view, and a +  icon
pops up that I can click and drag.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: 
***@***.***>

**Comment by Nostromos** [comment]
The problem I was having was either account or machine specific - neither Safari, Chrome, nor Firefox were working (both normal and incognito).

It's resolved itself in the last 5 minutes and seems to be working so I'll close this out.
